create-manifest.js

```diff
@@ -49,4 +38,22 @@ try {
   // demo dir (examples don't exist)
 }
 
-fs.writeFileSync('./dist/manifest.json', JSON.stringify(manifest, null, 2), { encoding: 'utf8' });
+/* add versioned bundles for CDN */
+function dirWalk (dir) {
+  const files = [];
+  function walk (dir) {
+    fs.readdirSync(dir).forEach(file => {
+      const absolute = path.join(dir, file);
+      if (fs.statSync(absolute).isDirectory()) return walk(absolute);
+      else files.push(absolute);
+    });
+    return files;
+  }
+
+  return walk(dir);
+}
+
+[...dirWalk(versionDir), ...dirWalk(majorVersionDir)]
+  .forEach(filename => manifest.indexFiles.push({ file: filename.replace('dist', '') }));
+
+fs.writeFileSync('./dist/manifest.json', JSON.stringify(manifest, null, 2), { encoding: 'utf8' });
```

Comment:
We will now only add `dist/v6/` and `dist/v6.1.7/` cdn files to the index files.